src/oscar/core/loading.py

```diff
@@ -224,6 +235,15 @@ def get_model(app_label, model_name):
     registry not being ready yet.
     Raises LookupError if model isn't found.
     """
+    oscar_moved_model = MOVED_MODELS.get(app_label, None)
+    if oscar_moved_model:
+        if model_name.lower() in oscar_moved_model[1]:
+            original_app_label = app_label
+            app_label = oscar_moved_model[0]
+            warnings.warn(
+                'Model %s has recently moved from %s to the application %s, '
+                'please update your imports.' % (model_name, original_app_label, app_label),
+                DeprecationWarning, stacklevel=2)
```

Comment:
We should use `RemovedInOscar22Warning` here (need to define this in `oscar.utils.deprecation`).